test/e2e-tests.sh

```diff
@@ -79,29 +90,22 @@ ci_run && {
   (( failed )) && fail_test
 }
 
-
 go build github.com/tektoncd/cli/cmd/tkn
 
 tkn() {
  ./tkn "$@"
 }
 
-# run_test "list pipelines" tkn pipeline list
-
-# ### Skipped as the command fails
-# run_test "list pipelines in an empty namespace" \
-#   SKIP tkn pipeline list -n default
-
-run_test  "list pipelines" SKIP tkn pipeline list
-run_test  "describe pipeline" SKIP tkn pipeline describe foo
-run_test  "list pipelinerun" SKIP tkn pipelinerun list
-run_test  "describe pipelinerun" SKIP tkn pipelinerun describe foo
-run_test  "show logs" SKIP tkn pipelinerun logs foo
-run_test  "list task" SKIP tkn task list
-run_test  "show logs" SKIP tkn taskrun logs foo
-run_test  "list taskrun" SKIP tkn taskrun list
+# command before creation of resources
+run_test  "list pipelines" tkn pipeline list
+run_fail_test  "describe pipeline that does not exist" tkn pipeline describe foo
+run_test  "list pipelinerun" tkn pipelinerun list
```

Comment:
also need 
`run_fail_test "list pipelinerun for pipeline foo that doesn't not exist" tkn pipelinerun list foo`?

Reply:
same behavior for `taskrun list` is applicable

Reply:
Added.